recipes/ty/conda_build_config.yaml

```
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge alpha
```

Labels should always include the package.
```suggestion
  - conda-forge ty_alpha
```